reactor-core/src/test/java/reactor/core/scheduler/DelegateServiceSchedulerTest.java

```diff
@@ -49,6 +50,31 @@ protected boolean shouldCheckDisposeTask() {
 		return false;
 	}
 
+	@Override
+	protected boolean shouldCheckSupportRestart() {
+		return false;
+	}
+
+	@Test
+	public void startAndDecorationImplicit() {
+		AtomicInteger decorationCount = new AtomicInteger();
+		Schedulers.setExecutorServiceDecorator("startAndDecorationImplicit", (s, srv) -> {
+			decorationCount.incrementAndGet();
+			return srv;
+		});
+		final Scheduler scheduler = new DelegateServiceScheduler("startAndDecorationImplicitExecutorService", Executors.newSingleThreadExecutor());
+
+		try {
+			assertThat(decorationCount).as("before schedule").hasValue(0);
+			scheduler.schedule(() -> {});
+			assertThat(decorationCount).as("after schedule").hasValue(1);
```

Comment:
I'd also add a second invocation of `schedule` and ensure that the count remains `1`

Reply:
done, I polished the tests and it should be stable now